config/routes.rb

```diff
@@ -0,0 +1,30 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :inventories, only: [:index, :show, :new, :create, :destroy]
+  resources :inventory_foods, only: [:create,:destroy]
+  resources :foods, only: [:new, :create, :destroy]
```

Comment:
![image](https://github.com/kensteph/Recipe-Group-project/assets/79058364/a67777b9-32f8-4bc5-bd5b-60be302674ac)

- [ ] Kindly let's address this by adding foods index route. 👍🏻